tests/fixture/tmpnet/utils.go

```diff
@@ -17,6 +20,29 @@ const (
 	DefaultNodeTickerInterval = 50 * time.Millisecond
 )
 
+func CheckNodeHealth(ctx context.Context, uri string) (*health.APIReply, error) {
```

Comment:
the most common error that I'd expect here is "context timeout". Would that be recoverable ?